chapter_4/chapter_4.md

```diff
@@ -0,0 +1,21 @@
+# Chapter 4:
+- General approach for building a model:
+1- Recognize the set of variables we are going to work with. The observable ones are called data, the unobservable ones, such as means or rates, are called parameters.
+2- Each variable is defined in terms of a probability distribution or in terms of other variables.
+3- The combination of variables and their probability distributions defines what it called a *joint generative model*. It can be used to simulate hypothetical observations or to analyze real ones.
+- The Gaussian is a member of a family of fundamental natural distributions known as the exponential family. All of the members of this family are important for working science, because they populate our world.
+- Using Gaussian distributions:
+    - Ontological justification: the world is full of Gaussian distributions, approximately.
```

Comment:
"the world is full of Gaussian distributions, approximately."
hmm I don't know if this sounds right. Maybe: "the world is full of phenomena that can be approximated with Gaussian distributions".